packages/suite/src/views/wallet/coinmarket/exchange/components/ExchangeForm/Inputs/SendCryptoInput/SendCryptoSelect/index.tsx

```diff
@@ -62,18 +62,20 @@ const SendCryptoSelect = () => {
                         setAmountLimits(undefined);
                         setValue(CRYPTO_INPUT, '');
                         setValue(FIAT_INPUT, '');
+                        setValue('outputs.0.address', account.descriptor);
                         const token = selected.value;
                         const invitySymbol = invityApiSymbolToSymbol(token).toLowerCase();
                         const tokenData = tokens?.find(t => t.symbol === invitySymbol);
                         if (ethereumTypeNetworkSymbols.includes(token)) {
```

Comment:
Please create another fork for solana networks, this solution makes it even more complicated than it is.

Remove setValue added on line 65 and do
If (ethereum)
else if (solana)
else

Reply:
https://github.com/trezor/trezor-suite/pull/11602/commits/0f5e6114ac1d80b9aa50e4977158bbaaabbb1f18